asv_bench/benchmarks/series_methods.py

```diff
@@ -59,21 +72,25 @@ def time_isin_empty(self):
 
 
 class IsInFloat64:
-    def setup(self):
-        self.small = Series([1, 2], dtype=np.float64)
+
+    params = [np.float64, "Float64"]
+    param_names = ["dtype"]
```

Comment:
use 5 and 6 for the powers rather than 6 and 7

you can factor these to the class as N_many, N_few

Reply:
Done.

Reply:
@jreback is there a reason to reduce the number? (on my laptop, the current takes around 80ms, which is fine I think?)

Reply:
the originals take multiple seconds which is very wasteful
10x less is appropriate

Reply:
The 80ms is what I get for the above code on current master. Where does the "multiple seconds" comes from?

Reply:
look at the benchmarks above

Reply:
If you mean the comment at https://github.com/pandas-dev/pandas/pull/38379#issuecomment-757258013, then the specific benchmark I am commenting on (`IsInFloat64`) is in that table already at lower values for some, eg:
```
-        96.4±2ms       76.8±0.5ms     0.80  series_methods.IsInFloat64.time_isin_many_different('Float64')
```

and for the others it is around 60ms after this PR, which seems like a fine number. 
So similar to the 80ms I mentioned that I get on master for the default float64 dtype.

It are other benchmarks (eg `IsInLongSeriesLookUpDominates`) that take more time. But those are not changed (yet) in this PR.

Reply:
pls read my comments

some of those benchmarks take 2 or more s

these all need to be normalized

Reply:
Then can you point me to the comment I might have missed? 
I don't contest that some benchmarks took seconds, the only thing I am saying is that *these* benchmarks I am commenting on (IsInFloat64) *do not*. 
If you don't believe me, you can take a look at https://pandas.pydata.org/speed/pandas/#series_methods.IsInFloat64.time_isin_few_different, https://pandas.pydata.org/speed/pandas/#series_methods.IsInFloat64.time_isin_many_different, https://pandas.pydata.org/speed/pandas/#series_methods.IsInFloat64.time_isin_nan_values. They are all in the 60-100ms range.

Reply:
look at the posted benchmarks